api/public_api_v1.yaml

```diff
@@ -4,40 +4,48 @@ info:
   title: Digital Citizenship API
   description: Digital Citizenship API.
 host: localhost
-basePath: /api/v1
+basePath: "/api/v1"
 schemes:
-  - https
+- https
 paths:
-  '/messages/{id}':
+  "/messages/{fiscal_code}/{id}":
     parameters:
-      - name: id
-        in: path
-        type: string
-        required: true
-        description: The ID of the message.
+    - name: fiscal_code
+      in: path
+      type: string
+      required: true
+      maxLength: 16
+      minLength: 16
+      pattern: '[A-Z]{6}[0-9LMNPQRSTUV]{2}[ABCDEHLMPRST][0-9LMNPQRSTUV]{2}[A-Z][0-9LMNPQRSTUV]{3}[A-Z]'      
+    - name: id
+      in: path
+      type: string
+      required: true
+      description: The ID of the message.
     get:
       operationId: getMessage
       summary: Get Message
-      description: The previously created message with the provided message ID is returned.
+      description: The previously created message with the provided message ID is
+        returned.
       responses:
         '200':
           description: Message found.
           schema:
-            $ref: '#/definitions/MessageResponse'
+            "$ref": "#/definitions/MessageResponse"
           examples: {}
         '404':
           description: No message found for the provided ID.
-  '/users/{fiscal_code}/messages':
+  "/messages/{fiscal_code}":
     parameters:
-      - $ref: '#/parameters/PaginationRequest'
-      - name: fiscal_code
-        in: path
-        type: string
-        required: true
+    - "$ref": "#/parameters/PaginationRequest"
+    - name: fiscal_code
+      in: path
+      type: string
+      required: true
         description: 'The fiscal code of the user, all upper case.'
```

Comment:
there's an extra indentation here, `description`, `maxLength`, `minLength` and `pattern` must be aligned with `required`

Reply:
besides, is it possible to reference `#/definitions/FiscalCode` here?